release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1535,6 +1535,8 @@ With {product-title} {product-version}, the `inspector.ipxe` configuration has b
 
 * Previously, the termination handler prematurely exited before marking a node for termination. This condition occurred based on the timing of when the termination signal was received by the controller. With this release, the possibility of early termination is accounted for by introducing an additional check for termination. (link:https://issues.redhat.com/browse/OCPBUGS-2117[*OCPBUGS-2117*])
 
+* Previously, when the `Build` cluster capability was not enabled, the cluster version Operator (CVO) failed to synchronize the build informer, and did not start successfully. With this release, the CVO successfully starts when the `Build` capability is not enabled. (link:https://issues.redhat.com/browse/OCPBUGS-22956[*OCPBUGS-22956*])
```

Comment:
I'm unsure if the third clause relates to the first or second clause? Perhaps the sentence should state (assumes the CVO did not start):

> Previously, when the `Build` cluster capability was not enabled, the cluster version Operator (CVO) failed to synchronize the build informer and did not start successfully. With this release, the CVO successfully starts when the `Build` capability is not enabled.

Other possibility:

> Previously, when the `Build` cluster capability was not enabled, the capability did not start because the Cluster Version Operator (CVO) failed to synchronize the build informer.

cluster version Operator/ Cluster Version Operator as per [existing docs](https://docs.openshift.com/container-platform/4.15/operators/operator-reference.html)?